sdk/webpubsub/Microsoft.Azure.WebPubSub.AspNetCore/src/Microsoft.Azure.WebPubSub.AspNetCore.csproj

```diff
@@ -13,7 +13,6 @@
   <ItemGroup>
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="System.Text.Json" />
-    <PackageReference Include="System.Text.Encodings.Web" VersionOverride="4.7.2"/>
```

Comment:
You need just a `<PackageReference Include="System.Text.Encodings.Web"/>` here because you don't depend on Azure.Core. Check your project.assets.json file to be sure after building again.

Reply:
I had tried that earlier. Since the project does not depend on `Azure.Core`, I get a package downgrade error.
C:\gh\azure-sdk-for-net-3\sdk\webpubsub\Microsoft.Azure.WebPubSub.Common\src\Microsoft.Azure.WebPubSub.Common.csproj : error NU1605: Detected package downgrade: System.Text.Encodings.Web from 4.6.0 to 4.5.1. Reference the package directly from the project to select a different version.